release_util/management/commands/__init__.py

```diff
@@ -131,165 +134,241 @@ def state(self):
         """
         return yaml.safe_dump(self.migration_state)
 
+def dump_migration_session_state(raw):
+    class Output(str): pass
+    class SessionDumper(yaml.SafeDumper): pass
+    def str_block_formatter(dumper, data):
+        return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='|')
+    SessionDumper.add_representer(Output, str_block_formatter)
+
+    raw = deepcopy(raw)
+    for step in raw:
+        step['output'] = Output(step['output'])
+    return yaml.dump(raw, Dumper=SessionDumper)
```

Comment:
If you look at SessionDumper, you'll see it inherits from SafeDumper. If you look at what `yaml.safe_dump` does, it just calls `yaml.dump(stuff, Dumper=SafeDumper)`. TLDR: this is equivalent to `yaml.safe_dump` with some extra formatting.